### PR TITLE
Initial Vanilla v1 pass on /index, /services, /products

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Covert /index, /products & /services to Vanilla v1

## QA

- Pull code
- Sanity check code
- Run `./run serve --watch`
- Check pages:

http://0.0.0.0:8002/
http://0.0.0.0:8002/products
http://0.0.0.0:8002/services
http://0.0.0.0:8002/services/contact-us
http://0.0.0.0:8002/services/thank-you

## Issue / Card

Related #144

### CODE REVIEW

**Comment by anthonydillon** [comment]
Is this ready for review? Missing labels.

File: templates/500.html [hunk 1]
```diff
@@ -150,26 +150,26 @@ <h2 class="footer__heading">Follow us</h2>
                     </li>
                 </ul>
             </div>
-            <div class="three-col last-col toggle-menu">
-                <h2 class="footer__heading--external find-out-more toggle-menu__button">Find out more</h2>
-                <ul class="external-link-list toggle-menu__content">
-                    <li class="external-link-list__list-item toggle-menu__list-item">
-                        <a class="external-link-list__link" href="https://www.ubuntu.com">Ubuntu</a>
+            <div class="col-3 toggle-menu">
+                <h2 class="footer__heading--p-link--external find-out-more toggle-menu__button">Find out more</h2>
```
Comment (anthonydillon) [inline templates/500.html]: space before external link

File: templates/500.html [hunk 2]
```diff
@@ -150,26 +150,26 @@ <h2 class="footer__heading">Follow us</h2>
...
+                <ul class="p-link--external-link-list toggle-menu__content">
```
Comment (anthonydillon) [inline templates/500.html]: p-link--external-link-list?

File: templates/_base/base.html [hunk 3]
```diff
@@ -1,130 +1,112 @@
...
+
+  <script>
+    var isOperaMini = (navigator.userAgent.indexOf('Opera Mini') > -1);
+    if(isOperaMini) {
+      var root = document.documentElement;
+      root.className += " opera-mini";
+    }
+  </script>
+
+  <script src="{{ STATIC_URL }}js/core.js"></script>
```
Comment (anthonydillon) [inline templates/_base/base.html]: Didn't you remove this file?

File: templates/_base/base.html [hunk 4]
```diff
@@ -1,130 +1,112 @@
...
+
+  <script>
+
+    // Every page should show the cookie policy until accepted
+    var cookie_options = {
+      'content': 'We use cookies to improve your experience of ubuntu.com. By continuing to explore without changing your settings, you are agreeing to accept them. To learn how to change these settings, please see our <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies">privacy policy</a>.'
```
Comment (anthonydillon) [inline templates/_base/base.html]: s/ubuntu.com/canonical.com

File: templates/_base/cookie-policy.html [hunk 5]
```diff
@@ -1,6 +1,5 @@
-<div class="cookie-policy">
-	<div class="wrapper">
-		<a href="?cp=close" class="link-cta">Close</a>
-		<p>We use cookies to improve your experience of ubuntu.com. By continuing to explore without changing your settings, you are agreeing to accept them. To learn how to change these settings, please see our <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies" class="external">privacy policy</a>.</p>
-	</div>
+<div class="p-notification">
+  <p class="p-notification__response">
+    We use cookies to improve your experience of ubuntu.com. By continuing to explore without changing your settings, you are agreeing to accept them. To learn how to change these settings, please see our <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies" class="p-link--external">privacy policy</a><a href="#" class="p-notification__action">Dismiss</a>.
```
Comment (anthonydillon) [inline templates/_base/cookie-policy.html]: This is using two versions of cookie policy and global nav. We should use the new version if possible.

File: templates/about/index.html [hunk 6]
```diff
@@ -13,205 +12,203 @@
 
 {% block content %}
 <div class="row row-hero">
-  <div class="inner-wrapper">
-    <div class="eight-col">
+  <div class="row">
```
Comment (anthonydillon) [inline templates/about/index.html]: Double `row` here.

File: templates/about/index.html [hunk 7]
```diff
@@ -13,205 +12,203 @@
 
 {% block content %}
 <div class="row row-hero">
```
Comment (anthonydillon) [inline templates/about/index.html]: Remove row-hero

File: templates/about/index.html [hunk 8]
```diff
@@ -13,205 +12,203 @@
...
+    <div class="col-4 text-center">
       <img src="{{ ASSET_SERVER_URL }}63127ce2-hero-about-small.png" alt="" />
-    </div><!-- /col -->
-    <div class="six-col">
-      <p class="intro">Canonical was created alongside Ubuntu to help it reach a wider market. Our services help governments and businesses the world over with migrations, management and support for their Ubuntu deployments. Together with our partners, we ensure that Ubuntu runs reliably on every platform from the PC and the smartphone to the server and, crucially, the&nbsp;cloud.</p>
-    </div><!-- /col -->
-  </div><!-- /inner-wrapper -->
-</div><!-- /row -->
+    </div>
+    <div class="col-6">
```
Comment (anthonydillon) [inline templates/about/index.html]: This will need a new row?

File: templates/about/index.html [hunk 9]
```diff
@@ -13,205 +12,203 @@
...
+      <p>Canonical was created alongside Ubuntu to help it reach a wider market. Our services help governments and businesses the world over with migrations, management and support for their Ubuntu deployments. Together with our partners, we ensure that Ubuntu runs reliably on every platform from the PC and the smartphone to the server and, crucially, the&nbsp;cloud.</p>
+    </div>
+  </div>
+</div>
 
-<div class="strip-dark row med-six-col">
-  <div class="inner-wrapper">
+<div class="strip-dark row">
```
Comment (anthonydillon) [inline templates/about/index.html]: Remove row please

File: templates/about/index.html [hunk 10]
```diff
@@ -13,205 +12,203 @@
...
                 <img src="{{ ASSET_SERVER_URL }}5a381903-icon-openstack.svg" class="icon" alt="" />
                 <p>We maintain a world class commercial partner network  around Ubuntu, including leading names such as VMware and world-changing open source technologies like OpenStack.</p>
             </li>
         </ul>
-    </div><!-- /inner-wrapper -->
-</div><!-- /row -->
+    </div>
+</div>
 
 <div class="row strip-light">
```
Comment (anthonydillon) [inline templates/about/index.html]: Remove row please

File: templates/about/index.html [hunk 11]
```diff
@@ -13,205 +12,203 @@
...
+    <div class="text-center col-9">
       <img src="{{ ASSET_SERVER_URL }}45888d3f-image-picto-projects-we-love.svg" width="195" height="196" alt="Heart pictogram" class="priority-0" />
-    </div><!-- /col -->
-  </div><!-- /inner-wrapper -->
-</div><!-- /row -->
+    </div>
+  </div>
+</div>
 
 <div class="row join">
```
Comment (anthonydillon) [inline templates/about/index.html]: remove row please

File: templates/about/index.html [hunk 12]
```diff
@@ -13,205 +12,203 @@
...
-  <div class="inner-wrapper">
-    <div class="twelve-col no-margin-bottom">
+  <div class="row">
+    <div class="col-12 no-margin-bottom">
       <h2>Our management team</h2>
     </div>
-    <div class="three-col no-margin-bottom">
+    <div class="col-3 no-margin-bottom">
```
Comment (anthonydillon) [inline templates/about/index.html]: remove no-margin-bottom

File: templates/about/index.html [hunk 13]
```diff
@@ -13,205 +12,203 @@
...
       <div class="tabbed-menu">
-        <ul class="no-bullets">
-          <li><a class="active slideless" href="#jane-silber">Jane Silber</a></li>
-          <li><a class="slideless" href="#mark-shuttleworth">Mark Shuttleworth</a></li>
+        <ul class="p-list">
+          <li><a class="active slideless" href="#mark-shuttleworth">Mark Shuttleworth</a></li>
```
Comment (anthonydillon) [inline templates/about/index.html]: Dont think we need slideless as thats controlled by js which has been removed.

File: templates/about/index.html [hunk 14]
```diff
@@ -13,205 +12,203 @@
...
         </div>
       </div>
     </div>
-    <div id="anand-krishnan" class="tabbed-content panel nine-col last-col hide" itemscope itemtype="http://schema.org/Person">
+    <div id="anand-krishnan" class="tabbed-content panel col-9 hide" itemscope itemtype="http://schema.org/Person">
       <a class="accordion-button slideless" href="#anand-krishnan">Anand Krishnan</a>
-      <div class="nine-col">
-        <div class="five-col">
+      <div class="col-9">
+        <div class="col-9">
```
Comment (anthonydillon) [inline templates/about/index.html]: Double col'ing

File: templates/about/index.html [hunk 15]
```diff
@@ -13,205 +12,203 @@
...
     </div>
-    <div id="chris-kenyon" class="tabbed-content panel nine-col last-col hide" itemscope itemtype="http://schema.org/Person">
-      <a class="accordion-button slideless" href="#chris-kenyon">Chris Kenyon</a>
-      <div class="nine-col">
-        <div class="five-col">
+    <div id="neil-french" class="tabbed-content panel col-9 hide" itemscope itemtype="http://schema.org/Person">
+      <a class="accordion-button slideless" href="#neil-french">Neil French</a>
+      <div class="col-9">
+        <div class="col-9">
+          <div class="clearfix">
```
Comment (anthonydillon) [inline templates/about/index.html]: clearfix is not a vanilla class

File: templates/about/index.html [hunk 16]
```diff
@@ -13,205 +12,203 @@
...
+            <img itemprop="image" class="portrait" src="{{ ASSET_SERVER_URL }}e1a539f7-neil-french.png" alt="" />
+            <h3 itemprop="name">Neil French</h3>
+            <p itemprop="jobTitle" class="role">VP, Operations</p>
+          </div><!-- /clearfix -->
+          <p itemprop="description">Neil is responsible for all of Canonical&rsquo;s internal operations.  Before joining Canonical, he was the Director of Commercial Operations at Thomsons Online Benefits, the leading Software-as-a-Service provider for global employee benefits administration, and before that held business and product leadership roles at Epson, 3Com and Hewlett-Packard.</p>
+          <p><a href="https://www.linkedin.com/in/neilfrench" class="external">View Neil on LinkedIn</a></p>
```
Comment (anthonydillon) [inline templates/about/index.html]: p-link--external

File: templates/about/index.html [hunk 17]
```diff
@@ -13,205 +12,203 @@
...
+        </div>
+        <div class="col-4">
+          <blockquote class="pull-quote">
```
Comment (anthonydillon) [inline templates/about/index.html]: Use vanilla pattern please

File: templates/about/index.html [hunk 18]
```diff
@@ -519,79 +516,79 @@ <h3 class="title">Isle of Man</h3>
 </div><!-- /map-address -->
 
 <div id="contact-row" class="row">
```
Comment (anthonydillon) [inline templates/about/index.html]: remove row please

File: templates/about/index.html [hunk 19]
```diff
@@ -519,79 +516,79 @@ <h3 class="title">Isle of Man</h3>
...
-  <div class="inner-wrapper">
-    <div class="four-col">
+  <div class="row">
+    <div class="col-4">
       <blockquote class="quote quote-left-bottom">
```
Comment (anthonydillon) [inline templates/about/index.html]: Use vanilla pattern please

File: templates/about/index.html [hunk 20]
```diff
@@ -519,79 +516,79 @@ <h3 class="title">Isle of Man</h3>
...
 
   <div class="row strip-light">
-    <div class="inner-wrapper">
-      <div class="twelve-col">
+    <div class="row">
+      <div class="col-12">
         <h2>Further information</h2>
       </div>
-    <div class="four-col">
+    <div class="col-4">
```
Comment (anthonydillon) [inline templates/about/index.html]: This will need a new row

File: templates/about/index.html [hunk 21]
```diff
@@ -519,79 +516,79 @@ <h3 class="title">Isle of Man</h3>
...
       <h3>Telephone enquiries</h3>
       <p>Our main switchboard number is +44&nbsp;20&nbsp;7630&nbsp;2400.</p>
-    </div><!-- /col -->
-  </div><!-- /inner-wrapper -->
-</div><!-- /row -->
+    </div>
+  </div>
+</div>
 
   <div class="row strip-light">
```
Comment (anthonydillon) [inline templates/about/index.html]: Remove row please

File: templates/about/index.html [hunk 22]
```diff
@@ -519,79 +516,79 @@ <h3 class="title">Isle of Man</h3>
...
         <p>On the Canonical blog, you&rsquo;ll find detailed analysis and personal viewpoints on Canonical announcements.</p>
       </div>
 
-    </div><!-- /inner-wrapper -->
-  </div><!-- /row -->
+    </div>
+  </div>
 
   <div class="row row-contextual-footer">
     <div class="inner-wrapper vertical-divider">
```
Comment (anthonydillon) [inline templates/about/index.html]: Not using vanilla class names here

File: templates/careers/all-design-and-user-experience-vacancies.html [hunk 23]
```diff
@@ -2,18 +2,17 @@
 
 {% block title %}Canonical | Careers | All Design and User Experience vacancies{% endblock %}
 {% block meta_description %}If you’re interested in open source software, the cloud or devices, Canonical is an incredible place to work. Whether you’re a technologist, a designer or a business professional, opportunities exist to work from home or at one of our key offices around the world.{% endblock %}
-{% block meta_keywords %}Canonical, Ubuntu, Linux, open source, free, software, design, designer, development, developer, programmer, engineer, UX, user, experience, interface, technology, IT, job, career, work, vacancy, vacancies, London, Taipei, Montreal, Boston, Shanghai, work from home, working from home, cloud, phone, smartphone, mobile{% endblock %}
 
 {% block content %}
 <div class="strip row">
```
Comment (anthonydillon) [inline templates/careers/all-design-and-user-experience-vacancies.html]: Remove row

File: templates/careers/all-design-and-user-experience-vacancies.html [hunk 24]
```diff
@@ -2,18 +2,17 @@
...
+        <div class="col-8">
             <h1>All Design and User Experience vacancies</h1>
         </div>
-        <div class="eight-col">
-            <p class="intro">If you are interested in a role in a specific department or location, you can <a href="https://ldd.tbe.taleo.net/ldd03/ats/careers/jobSearch.jsp?org=CANONICAL&cws=1&rid=86">search jobs</a>.</p>
+        <div class="col-8">
+            <p>If you are interested in a role in a specific department or location, you can <a href="https://ldd.tbe.taleo.net/ldd03/ats/careers/jobSearch.jsp?org=CANONICAL&cws=1&rid=86">search jobs</a>.</p>
         </div>
-        <div class="eight-col">
+        <div class="col-8">
```
Comment (anthonydillon) [inline templates/careers/all-design-and-user-experience-vacancies.html]: Missing new rows

File: templates/careers/all-marketing-and-business-development-vacancies.html [hunk 25]
```diff
@@ -2,24 +2,23 @@
...
-				{% get_rss_feed "http://ldd.tbe.taleo.net/ldd03/ats/servlet/Rss?org=CANONICAL&cws=1&_rss_version=2&CUSTOM_755=11588,6158,616" as vacancies %}
-				{% include "_includes/vacancies.html" with vacancies=vacancies %}
-			</ul>
-		</div>
-	</div>
+  <div class="row">
+    <div class="col-8">
+      <h1>All Marketing and Business Development vacancies</h1>
+    </div>
+    <div class="col-8">
```
Comment (anthonydillon) [inline templates/careers/all-marketing-and-business-development-vacancies.html]: This needs more rows

File: templates/careers/cloud-vacancies.html [hunk 26]
```diff
@@ -2,25 +2,24 @@
...
+    <div class="row">
+        <div class="col-6">
             <h1>All cloud vacancies</h1>
             <p>No other operating system can compete with Ubuntu’s long history of integration with the open cloud – and we are always working to widen that lead. Our cloud teams work on MaaS, our provisioning software, and Juju, our service orchestration tool – not to mention OpenStack itself.
 In short, we’re all about cloud. Are you?</p>
-            <p><a href="https://www.ubuntu.com/cloud" class="external">More about our cloud products</a></p>
+            <p><a href="https://www.ubuntu.com/cloud" class="p-link--external">More about our cloud products</a></p>
         </div>
-        <div class="six-col last-col text-center">
+        <div class="col-6 text-center">
```
Comment (anthonydillon) [inline templates/careers/cloud-vacancies.html]: text-center is not a vanilla class

File: templates/careers/go-vacancies.html [hunk 27]
```diff
@@ -2,26 +2,25 @@
...
+            <p><a href="https://code.launchpad.net/juju-core" class="p-link--external">Check out the code</a></p>
         </div>
-        <div class="push-one five-col last-col">
+        <div class="push-one col-5">
             <img src="{{ ASSET_SERVER_URL }}e930b8ae-image-govacancies.svg" alt="Golang Gopher">
         </div>
     </div>
 </div>
 
 <div class="row strip strip-light">
```
Comment (anthonydillon) [inline templates/careers/go-vacancies.html]: remove row and use p-strip

File: templates/careers/go-vacancies.html [hunk 28]
```diff
@@ -2,26 +2,25 @@
...
-    <div class="inner-wrapper">
+    <div class="row">
         <div class="four-col profile">
```
Comment (anthonydillon) [inline templates/careers/go-vacancies.html]: Old grid classes

File: templates/careers/index.html [hunk 29]
```diff
@@ -12,39 +11,39 @@
...
-            <p class="intro"><a class="see-all" href="/careers/all-vacancies">See all vacancies &rsaquo;</a></p>
+        <div class="col-6">
+            <p>Life at Canonical is anything but corporate. As a company that exists to support one of today&rsquo;s most important open source projects, we are changing the world on a daily basis. It&rsquo;s a collaborative environment, but one in which every member of the team takes personal responsibility for everything they&nbsp;produce.</p>
+            <p><a class="see-all" href="/careers/all-vacancies">See all vacancies &rsaquo;</a></p>
         </div>
     </div>
 </div>
-<div class="row med-six-col strip-dark">
-    <div class="inner-wrapper">
+<div class="row med-col-6 strip-dark">
```
Comment (anthonydillon) [inline templates/careers/index.html]: med-col-6 is not a thing

Comment (anthonydillon) [inline templates/careers/index.html]: please use p-strip

File: templates/careers/index.html [hunk 30]
```diff
@@ -12,39 +11,39 @@
...
+    <div class="row">
         <h2>Reasons to join us at Canonical</h2>
         <ul class="connected-list">
-            <li class="three-col">
+            <li class="col-3">
                 <img src="{{ ASSET_SERVER_URL }}80a8e84c-icon-flash.svg" class="icon" alt="">
```
Comment (anthonydillon) [inline templates/careers/index.html]: Please use the p-heading-icon pattern

File: templates/careers/index.html [hunk 31]
```diff
@@ -74,15 +73,15 @@
             </p>
             <img src="{{ ASSET_SERVER_URL }}1000a177-tabbed-nav-arrow.png" class="arrow" height="6" width="12" alt="" />
         </div>
-        <div id="design-and-user-experience" class="tabbed-content panel nine-col last-col">
+        <div id="design-and-user-experience" class="tabbed-content panel nine-col">
```
Comment (anthonydillon) [inline templates/careers/index.html]: old grid class

File: templates/careers/index.html [hunk 32]
```diff
@@ -74,15 +73,15 @@
...
             <a class="accordion-button slideless" href="#design-and-user-experience">Design and User Experience</a>
             <div>
                 <div class="nine-col vertical-divider no-margin-bottom">
                     <h3 class="title">Design and User Experience</h3>
-                    <div class="five-col">
+                    <div class="col-5">
                         <p>There are few organisations in the world where interface designers, graphic designers and user experience professionals come together on such a fascinating array of projects. From our office in London, this team creates the look, feel and interaction models for technologies as diverse as the Juju service orchestration tool and the Ubuntu Touch mobile interface. </p>
                     </div>
-                    <div class="four-col last-col">
+                    <div class="four-col">
```
Comment (anthonydillon) [inline templates/careers/index.html]: old grid class

File: templates/careers/index.html [hunk 33]
```diff
@@ -92,15 +91,15 @@ <h3 class="title">Design and User Experience</h3>
...
+                    <div class="col-3">
                         <img class="portrait" itemprop="image" src="{{ ASSET_SERVER_URL }}acbdf79d-portrait-rosie.png" alt="">
                         <ul class="personal-info">
                             <li class="name" itemprop="name">Xi Zhu</li>
                             <li class="role" itemprop="jobTitle">Visual Designer</li>
                             <li class="location">London</li>
                         </ul>
                     </div>
-                    <div class="six-col last-col no-margin-bottom">
+                    <div class="col-6 no-margin-bottom">
```
Comment (anthonydillon) [inline templates/careers/index.html]: Remove no-margin-bottom

File: templates/careers/index.html [hunk 34]
```diff
@@ -110,15 +109,15 @@ <h3 class="title">Design and User Experience</h3>
                 </div>
             </div>
         </div>
-        <div id="technical-and-engineering" class="tabbed-content panel nine-col last-col hide">
+        <div id="technical-and-engineering" class="tabbed-content panel nine-col hide">
```
Comment (anthonydillon) [inline templates/careers/index.html]: Lots of old grid classes

**Comment by barrymcgee** [comment]
@anthonydillon You reviewed the whole site - only /index, /services, /products have had first pass as per PR title.
